src/Enterspeed.Source.Sdk/Domain/Services/EnterspeedIngestService.cs

```diff
@@ -142,7 +146,7 @@ public Response Delete(string id)
                     .GetAwaiter()
                     .GetResult(),
                 Status = response.StatusCode,
-                Success = response.StatusCode == HttpStatusCode.OK
+                Success = response.IsSuccessStatusCode
```

Comment:
Particular line concerns me, we used to compare to OK, but according to some of our swagger definitions and some logic, it could be something else rather than 200, eg. 201, 204 etc.

I don't think 200, must be a blocker for success.

Reply:
@elglogins You are concerned for your own change? 😕

Reply:
I'm concerned about previous one

Reply:
@elglogins Ahh, got it :)

Reply:
Agree `IsSuccessStatusCode` is better